apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -31,7 +32,12 @@ export const ChatBubbleList = ({ posts, subToMore }: ChatBubbleListProps) => {
         chatDate={formatChatBubbleDate(new Date(post.createdTime))}
         userPicture={avatarUrl(post.user)}
       >
-        {post.message}
+        <>
+          {post.media && (
+            <img src={getMediaUrl(post.media.id)} className="chat-bubble-image" alt="" />
+          )}
+          {post.message !== '' && post.message}
+        </>
```

Comment:
💡 From discord, I'm used to the image being below the text. What do you think?

Reply:
I can argue that from IMessage, I'm used to text being at the bottom